playbooks/verify_autoscaling.yaml

```diff
@@ -42,4 +42,17 @@
         openstack metric status --fit-width;
       register: result
       failed_when: result.rc >= 1
+
+    - name: Check if flavor "small" exists
```

Comment:
Ideally, we should delete any flavors that we create. 
Either create the flavor in the heat template (if this is possible), or create the image (with a unique name) here that gets deleted after the test is done.

Reply:
I just wanted to check if it will solve the problem, and then we can enhance the solution

Reply:
That's fair.

We're planning on removing this repo anyway, once we have tested the FVT integration that will hopefully work for both tripleo- and openstack-k8s-operators- based installs